lib/holdings-updater.js

```diff
@@ -0,0 +1,295 @@
+'use strict'
+
+const H = require('highland')
+let log = require('loglevel')
+const cluster = require('cluster')
+const UpdaterBase = require('./updater-base')
+
+const HoldingSierraRecord = require('./models/holding-sierra-record')
+const db = require('./db')
+const utils = require('./utils')
+const IndexerRunner = require('./indexer-runner')
+const SourceLoader = require('./sources/loader')
+const holdingSerializer = require('./serializers/holding')
+const nyplApi = require('./nyplDataApi')
+const kmsHelper = require('./kms-helper')
+
+const CACHE_TO_DISK = false
+
+class HoldingsUpdater extends UpdaterBase {
+  deserialize (buffered, avro) {
+    let holding
+
+    try {
+      holding = avro.fromBuffer(buffered.value, 'Holding', buffered.offset)
+      log.info(`HoldingUpdater#deserialized ${buffered.offset}: ${holding.id}`)
+    } catch (e) {
+      log.error('__________ERROR__ Error decoding value: ', e)
+    }
+
+    return holding
+  }
+
+  saveToDisc (holding) {
+    const path = `./cache/holdings/${holding.id}.json`
```

Comment:
There is no `nyplSource` in the holdings record, so I've eliminated that wherever it appears. I don't think that will have any side effects. (Here for items it is `./cache/items/${nyplSource/i${item.id}.json`